Index: services/eat/src/main.rs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>use serde_json::Value;\n\n\n#[macro_use] extern crate rocket;\nextern crate common;\nuse common::google::Google;\n\n\n// url static\n#[get(\"/\")]\nasync fn index() -> String {\n    let resto = get_google().await;\n    format!(\"{:?}\", resto)\n}\n\n\n// ajout de variable dans l'url\n#[get(\"/hello/<name>\")]\nfn hello(name: &str) -> String {\n    format!(\"Hello, {}!\", name)\n}\n\n#[launch]\nfn rocket() -> _ {\n    rocket::build().mount(\"/\", routes![index, hello])\n}\n\n\nasync fn get_google() -> Value{\n    let mut google = Google::new();\n    google.geocoding(String::from(\"Paris\")).await.expect(\"TODO: panic message\");\n    let resto: Value = google.nearby_place(String::from(\"restaurant\"), 1000).await.expect(\"FDP\");\n    resto\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/services/eat/src/main.rs b/services/eat/src/main.rs
--- a/services/eat/src/main.rs	(revision be2d6dfd31ba806ad57a99690dea7fc96827e5dc)
+++ b/services/eat/src/main.rs	(date 1710851878085)
@@ -4,13 +4,15 @@
 #[macro_use] extern crate rocket;
 extern crate common;
 use common::google::Google;
+use common::google::nearly_place_model::exploit_json;
 
 
 // url static
 #[get("/")]
 async fn index() -> String {
     let resto = get_google().await;
-    format!("{:?}", resto)
+    let result = exploit_json(resto);
+    format!("{:?}", result)
 }
 
 
